src/pages/cplusplus/variable.md

```diff
@@ -0,0 +1,27 @@
+Lets discuss something know as variables. Variables are like a bucket. You can put something in it and then change it 
```

Comment:
Can you add the following YAML header above this line? Thanks!

```yaml
---
title: Variables
---
```

Reply:
The header has been added!